src/SweetPoll.elm

```diff
@@ -47,10 +46,10 @@ defaultConfig : Json.Decoder data -> String -> Config data
 defaultConfig decoder url =
     { decoder = decoder
     , url = url
-    , delay = 7 * Time.second
+    , delay = 1000 * 7
     , samesBeforeDelay = 3
     , delayMultiplier = 1.2
-    , maxDelay = 3 * Time.minute
+    , maxDelay = 1000 * 60 * 3
```

Comment:
I'm sad that `Time.$unit` has gone. I expect there was good reason... or is it simply waiting for someone to resurrect it in the new `elm/time`?